src/organization/screens/organization-profile.screen.js

```diff
@@ -84,23 +56,29 @@ class OrganizationProfile extends Component {
     };
   }
 
-  componentDidMount() {
-    const organization = this.props.navigation.state.params.organization;
+  componentWillMount() {
+    loadData(this.props);
+  }
 
-    this.props.fetchOrganizations(organization.login);
-    this.props.fetchOrganizationMembers(organization.login);
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.orgId !== this.props.orgId) {
+      loadData(nextProps);
+    }
   }
 
-  getOrgData = () => {
-    const organization = this.props.navigation.state.params.organization;
+  refreshData = () => {
+    const { navigation, getMembers, getById } = this.props;
+    const orgId = navigation.state.params.organization.login;
+
+    navigation.setParams({ refreshing: true });
+    getById(orgId, { forceRefresh: true });
+    getMembers(orgId, { forceRefresh: true });
+  };
```

Comment:
This can be a mutualized behavior in our screens, where we just call loadData() with force refresh